levelObjects/Actions.md

```diff
@@ -432,18 +442,26 @@ This change has no parameters (all 12 bytes 0x00) and lets the player receive th
 The game continues after the message has played. Images and text are hardcoded.
 
 
-#### Change State Type 16: Unknown
+#### Change State Type 16: Change type
 
     0000  int32      Object type - 0x00CCSSTT
-    0004  int32      Unknown
-    0008  int32      Unused
-
-> This entry is only used on level 8, for 4 buttons. Purpose and effect unknown.
-
+    0004  byte       New type
+    0008  [8]byte    Unused
 
-### Action Type 21: Unknown
+Changes item type. Class and subclass are not changed.
+	
+> This entry is only used on level 8, for 4 buttons.
 
 
+### Action Type 21: Activate critters
```

Comment:
Based on the new critter knowledge, I could determine this as "set critter state".